components/employer/EditNotice/EditNotice.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import { FormEvent, useState } from "react";
+import { CommonBtn, CustomInput } from "components/common";
+import { ButtonSize, ButtonStyle } from "types/enums/button.enum";
+import styles from "./EditNotice.module.scss";
+
+const EditNotice = () => {
```

Comment:
폼이란게 드러나게 작명하는건 어떨까요?
```suggestion
const EditNoticeForm = () => {
```